client/pages/ProfilePage.tsx

```diff
@@ -14,11 +93,46 @@ function ProfilePage() {
           login flow powered by Stytch. You can review the source code for this
           app on Github to learn how to implement this yourself!
         </Text>
-        <Text>
-          Your user ID is {!!user && user.user_id}
-        </Text>
+        <Text>Your user ID is {!!user && user.user_id}</Text>
+        <Text>Biometry type available: {sensorType}</Text>
       </View>
-      <TouchableOpacity style={[sharedStyles.buttonDark]} onPress={stytch.session.revoke}>
+      {isKeystoreAvailable && (
```

Comment:
Maybe `isKeystoreAvailable && !hasBiometricRegistration && (`?

Probably don't need to expose the registration override in the demo app?